base/debian-9/install.sh

```diff
@@ -46,7 +46,7 @@ ln -s busybox syslogd
 ln -s busybox tail
 ln -s busybox traceroute
 ln -s busybox vi
-
+chmod u+s /bin/ping
```

Comment:
May need to do this for ping6 as well - the other symlinks we created don't seem to be affected by this permission issue

Reply:
this is needed to create an icmp packet, ping6 doesn't actually use icmp in the routine.  I did test it, and ran through a few of the other commands.  I didn't see any others that failed.